EpikCord/__main__.py

```diff
@@ -1,4 +1,60 @@
-from EpikCord import __version__
+import argparse
+import colorama
+
+
+__all__ = ("__version__", "info")
+__version__ = "0.5.2"
+_bot_tmplate = """#!/usr/bin/env python
+from EpikCord import Client, Intents
+
+intents = Intents.all()
+
+client: Client = Client([token], intents)
+
+
+@client.event()
+async def ready():
+    print("Ready!")
+# Write commands here
+
+client.login()
+"""
+win_restricted = (
+    "CON",
+    "CON1",
+    "CON2",
+    "CON3",
+    "CON4",
+    "CON5",
+    "CON6",
+    "CON7",
+    "CON8",
+    "CON9",
+    "AUX",
+    "AUX1",
+    "AUX2",
+    "AUX3",
+    "AUX4",
+    "CON5",
+    "CON6",
+    "CON7",
+    "CON8",
+    "CON9",
+
+)
```

Comment:
```py
re.fullmatch(r'(AUX|CON)\d?', string)
```
...

Reply:
i did not commit new code yet

Reply:
there are more restricted words thatn that

>

Reply:
> there are more restricted words thatn that
> 
> >

yeah but a regex will do it, also that only apply to windows, so it's not like we such really care a lot

Reply:
well, if you are in a windows system than you **cannot** access the file and it might also render the system unusable

Reply:
> well, if you are in a windows system than you **cannot** access the file and it might also render the system unusable

i mean it pretty much guaranteed that the user is not dumb enough to ask bot a bot with a user reserved name as the directory...

Reply:
```py
re.fullmatch(r'((AUX|CON|LPT)\d?)|NUL', string)
```

Reply:
The average user dont know it btw